### PR TITLE
v6.2.0

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,140 @@
+<a name="6.2.0"></a>
+# 6.2.0 (2020-08-06)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.1.1...v6.2.0)
+
+We expect this release to ship in the DevTools of [Chrome 86](https://chromiumdash.appspot.com/schedule), and to PageSpeed
```
Comment (patrickhulce) [inline changelog.md:5]: look how much extra time we have in canary! 🎉  :D
image

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+* Stacey Tay @staceytay
+* Wojciech Maj @wojtekmaj
+* moli @phpmoli
+* Муравьёв Семён @Zulcom
+
+## New Audits
+
+* move duplicated-javascript and legacy-javascript to default config ([#10881](https://github.com/GoogleChrome/lighthouse/pull/10881), [#11103](https://github.com/GoogleChrome/lighthouse/pull/11103))
+* report animations not run on compositor ([#11105](https://github.com/GoogleChrome/lighthouse/pull/11105))
+* add unsized-images to experimental config ([#11188](https://github.com/GoogleChrome/lighthouse/pull/11188), [#11115](https://github.com/GoogleChrome/lighthouse/pull/11115), [#11217](https://github.com/GoogleChrome/lighthouse/pull/11217))
```
Comment (patrickhulce) [inline changelog.md:31]:
```suggestion
* add unsized-images audit ([#11188](https://github.com/GoogleChrome/lighthouse/pull/11188), [#11115](https://github.com/GoogleChrome/lighthouse/pull/11115), [#11217](https://github.com/GoogleChrome/lighthouse/pull/11217))
```

it got moved to default right?

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,140 @@
...
+* Pramod Mali @malipramod
+* Robin Tom @robintom
+* Stacey Tay @staceytay
+* Wojciech Maj @wojtekmaj
+* moli @phpmoli
+* Муравьёв Семён @Zulcom
+
+## New Audits
+
+* move duplicated-javascript and legacy-javascript to default config ([#10881](https://github.com/GoogleChrome/lighthouse/pull/10881), [#11103](https://github.com/GoogleChrome/lighthouse/pull/11103))
```
Comment (patrickhulce) [inline changelog.md:29]:
```suggestion
* move duplicated-javascript and legacy-javascript audits to default config ([#10881](https://github.com/GoogleChrome/lighthouse/pull/10881), [#11103](https://github.com/GoogleChrome/lighthouse/pull/11103))
```

File: lighthouse-core/scripts/release/bump-versions.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ for (const file of glob.sync('**/{package.json,*.md}', {ignore})) {
   let text;
   if (file === 'package.json') {
     const pkg = require(path.resolve(file));
+    if (pkg.version.startsWith('file')) continue;
```
Comment (patrickhulce) [inline lighthouse-core/scripts/release/bump-versions.js:24]: what was this fixing? comment?

File: changelog.md (hunk 3 above)
Comment (patrickhulce) [reply changelog.md:29]: idk feels like we should make a bigger deal by announcing these linking in their real PRs but they're your sweat and tears so up to you :)

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,138 @@
+<a name="6.2.0"></a>
+# 6.2.0 (2020-08-06)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.1.1...v6.2.0)
+
+We expect this release to ship in the DevTools of [Chrome 86](https://chromiumdash.appspot.com/schedule), and to PageSpeed
+Insights within 2 weeks.
+
+## New Contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻!
```
Comment (paulirish) [inline changelog.md:10]: check out these four pros at the top!  

WE SHIPPING, BOYSSSSS. :D

**Pull request merged**